RUMM-1611: Track clicks in Jetpack Compose (#757)
DataDog/dd-sdk-android: dd-sdk-android-compose/README.md

    .build()
```

#### Action tracking

> There is no automated instrumentation support for the action tracking in Jetpack Compose yet. However, you may report clicks using `trackClicks` method in the following manner:

Reviewer: ```suggestion
There is no automated instrumentation support for the action tracking in Jetpack Compose. However, report clicks using `trackClicks` method using this example:
```

```kotlin
Button(

>     onClick = trackClicks(targetName = "Open View") { ...click logic... }

Reviewer 2: I think the added value of this is the abstraction, which may enable us to do internal improvements later, without changing the `RUMMonitor` API 👌.

Author: true, good point.
